src/client/components/app-header/app-header.vue

```diff
@@ -40,23 +41,62 @@ export default {
 <style>
 @import '../app-core/index.css';
 
-.app-header__menu-link {
+.app-header {
+  width: 100%;
+  height: var(--app-header-height);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  position: sticky;
+  top: 0;
+  left: 0;
+  padding: 0 1rem;
+  background-color: #fff;
+  opacity: .9;
+  box-shadow: 0 2px 15px 0 rgba(214,214,214,.5);
+  z-index: 10000;
```

Comment:
i think a `z-index` of 1 is enough :) i get that for now you always want it to be on top but it will make it harder to keep track of stacking context when we start using values like this.